functions/src/util.ts

```diff
@@ -159,3 +160,30 @@ export function firebasePathEncode(s: string) {
 export function firebasePathDecode(s: string) {
   return s.replace(/_/g, '.');
 }
+
+export async function getJwtToken(clientEmail: string, privateKey: string): Promise<string> {
+
+  // Define the required scopes.
+  const scopes = [
+    "https://www.googleapis.com/auth/userinfo.email",
+    "https://www.googleapis.com/auth/firebase.database"
+  ];
+  const jwtClient = new google.auth.JWT(
+    clientEmail,
+    null,
+    privateKey,
+    scopes
+  );
+  
+  return new Promise<string>((resolve, reject) => {
+    jwtClient.authorize(function(error,  tokens) {
+      if (error) {
+        // reject(error);
```

Comment:
Why are you not rejecting here and below? You might create a promise that stays pending and never resolves

Reply:
fixed